build.sbt

```diff
@@ -4,22 +4,20 @@ scalacOptions ++= Seq("-deprecation", "-feature")
 
 organization := "de.leanovate.play-mockws"
 
-val playVersion = "2.8.1"
+val playVersion = "3.0.0"
 
 fork := true
 
-resolvers += "Typesafe repository".at("https://repo.typesafe.com/typesafe/releases/")
-
 libraryDependencies ++= Seq(
-  "com.typesafe.play"      %% "play"                    % playVersion % "provided",
-  "com.typesafe.play"      %% "play-ahc-ws"             % playVersion % "provided",
-  "com.typesafe.play"      %% "play-test"               % playVersion % "provided",
-  "org.scala-lang.modules" %% "scala-collection-compat" % "2.1.6"
+  "org.playframework"      %% "play"                    % playVersion % "provided",
+  "org.playframework"      %% "play-ahc-ws"             % playVersion % "provided",
+  "org.playframework"      %% "play-test"               % playVersion % "provided",
+  "org.scala-lang.modules" %% "scala-collection-compat" % "2.5.0"
```

Comment:
Now that you dropped Scala 2.12, this library is not needed anymore.
See the project's [README](https://github.com/scala/scala-collection-compat):
> This library makes some Scala 2.13 APIs available on Scala 2.11 and 2.12.

 Scala 2.13+ has this built-in now, you can replace code like:
```scala
import scala.collection.JavaConverters._
```
with
```scala
import scala.jdk.CollectionConverters._
```

@rtyley did the same in https://github.com/guardian/play-secret-rotation/pull/411

Reply:
Good catch, updated!